templates/catalog/options.html

```diff
@@ -4,14 +4,9 @@
 {% for product in products %}
   <tr class="table-tr" data-id="{{ product.id }}" data-name="{{ product.name }}">
     <td class="table-td table-photo text-center">
-      <b>-</b>
-      {% comment %}
-        Will be fixed with #565
-
-        {% with image=product_images|get_item:product.id %}
-          {% include 'catalog/option_image.html' with image=image page=product.page only %}
-        {% endwith %}
-      {% endcomment %}
+      {% with image=product_images|get_item:product.id %}
```

Comment:
It seems we should use here `product.product.id` as key for `product_images` dict, because these `products` are actually `options` and `product_images`'s keys are product ids, shouldn't we?

Reply:
@artemiy312 , sure, i'll fix it.

Test should fall, but didn't because i'm so lucky :D
```
def test_product_image_button(self):
    """Series page should contain button to open existing product image."""
    # product with image
    product = models.Product.objects.get(id=110)
    print([o.id for o in product.options.all()])
    response = self.client.get(product.category.url)
    self.assertContains(response, 'table-photo-ico')

# output
[110]
.
----------------------------------------------------------------------
Ran 1 test in 3.113s

OK
```